test/distributed/test_symmetric_memory.py

```diff
@@ -152,24 +152,27 @@ def test_empty_strided_p2p_persistent(self, set_device: bool) -> None:
         self._init_process(set_device)
         enable_symm_mem_for_group(dist.group.WORLD.group_name)
 
+        alloc_id = 42 + int(set_device)
```

Comment:
nit: we can make `alloc_id` a random number, in case `int(set_device)` looks weird